### ISSUE TITLE
BMI is incorrect

### ISSUE THREAD

**Issue by sanacl** [chaoss/grimoirelab-sigils#502]
The current calculation we do to offer the BMI is wrong according to [this paper](http://41.89.227.156:8080/xmlui/bitstream/handle/123456789/423/Assessing%20Quality%20of%20Open%20Source%20Software%20Based%20on%20Community_Metrics.pdf?sequence=1&isAllowed=y) (see page 8 of the pdf)

In the current calculation we are not multiplying by 100. This new value will help us to explain the result, as it shows the percentage of tickets you are being able to digest per week. So it is more clear that 90% is that you are not being able to handle all the tickets (in a period of time)

The current calculation of the Timeline/Timelion chart is below:
```
.es(index=github_issues, q='pull_request:false', timefield=closed_at).divide(.es(index=github_issues, q='pull_request:false', timefield=grimoire_creation_date)).label('BMI').color(#99af5d),
.es(index=github_issues, q='pull_request:false', timefield=closed_at).divide(.es(index=github_issues, q='pull_request:false', timefield=grimoire_creation_date)).movingaverage(8w, 'left').label('BMI Moving Avg. (8 weeks)').color(#c83d32),
.es(index=github_issues, q='pull_request:false', timefield=closed_at).divide(.es(index=github_issues, q='pull_request:false', timefield=grimoire_creation_date)).trend().label('BMI trend'),
.es(index=github_issues, q='pull_request:false', timefield=closed_at).divide(.es(index=github_issues, q='pull_request:false', timefield=grimoire_creation_date)).aggregate(avg).label('Avg. BMI').color(#e6b740)
```

**Comment by sduenas** [comment]
Is it really a bug or is just a problem about how users can understand this metric better?

**Comment by sanacl** [comment]
If you look for papers or even web sites where the Backlog Management Index is explained, the standard definition is:  ´(closed issues / opened issues) * 100`. This is **not** what we have in the GrimoireLab metrics

**Comment by alpgarcia** [comment]
Percentage means divided by 100. Percentage sign "%" then means divided by 100. Thus, 100% = 100 / 100 = 0,1.

It's not a bug, just two ways of expressing the same. If they multiply by 100 for any other reason, then it's a matter of scale.

**Comment by GeorgLink** [comment]
This means that when we close issues but don't open new issues, our BMI should be infinite!!! (breaking the chart :-D )

**Comment by sanacl** [comment]
Two questions for you:
 -  where is the BMI defined as `closed/open without being multiplied by 100`? If you think it should be `closed/open` then we need to publish this specification somewhere: blog post, research paper, whatever ..
 -  don't you think it is more easy to be understood, that if you get 100% you are closing the same amount of tickets you are getting?

**Comment by alpgarcia** [comment]
> * where is the BMI defined as `closed/open without being multiplied by 100`? If you think it should be `closed/open` then we need to publish this specification somewhere: blog post, research paper, whatever ..

As I said, 100% = 1. It's not any specification, it's math. If you specify to multiply by 100 and then you say the result is 100%, then you are saying the result is `100/100 = 1`.

> * don't you think it is more easy to be understood, that if you get 100% you are closing the same amount of tickets you are getting?

This is a different question and I'm not against it, I'm against to consider this a bug because it complies with the metric definition.

**Comment by canasdiaz** [comment]
I've found the place where the BMI was defined in 2002. The book is "Metrics and Models in Software Quality Engineering, 2nd Edition" By Stephen H. Kan

Find below the definition:
![imagen](https://user-images.githubusercontent.com/326719/233369543-8493417f-05c2-40e2-b4eb-3f9418d86bf1.png)

More info at https://www.informit.com/articles/article.aspx?p=30306&seqNum=3

**Comment by canasdiaz** [comment]
As @fioddor pointed out, the definition of the metric is not coherent. For one side, the factor is multiplied by a percentage. It does not make sense. Second, the author mentions the value without the percentage but multiplied by 100.

As a minimal consensus, I believe we can agree that the BMI must be displayed in percentage. Do you agree?

**Comment by alpgarcia** [comment]
Yes, I agree. Please, remove the bug label and add enhancement instead for clarity. One might think the metric is not displaying the right value when that's not the case, is just a matter of mathematical terminology.
